Support/PowershellSyntax.tmLanguage_

```diff
@@ -522,7 +522,7 @@
 		<key>function</key>
 		<dict>
 			<key>begin</key>
-			<string>((?i:function|filter|configuration|workflow))\s+((?:\p{L}|\d|_|-|\.)+)</string>
```

Comment:
I've messed this up. This change wasn't supposed to end up here. I'll try to fix it later.